config/src/test/kotlin/maru/config/HopliteFriendlinessTest.kt

```diff
@@ -66,7 +66,7 @@ class HopliteFriendlinessTest {
               endpoint = URI.create("http://localhost:8545").toURL(),
             ),
           qbftOptions = QbftOptions(100.milliseconds),
-          p2pConfig = P2P(port = 3322u),
+          p2pConfig = P2P(),
```

Comment:
But the port should be 3322, not 9000. Something is wrong with this test. Also, we need to test that the rest of the P2P configs are configurable

Reply:
Ok, I can actually see the test is failing because of that